lib/tool_mount_point/wpcc.rb

```diff
@@ -0,0 +1,6 @@
+module ToolMountPoint
+  class Wpcc < Base
+    EN_ID = 'wpcc'
```

Comment:
Even if's not stated, by intuition, we should be able to say, "_lets do this properly_." These IDs will need to be changed with another story at another date just before the launch. Is it not better to do it in this PR? Especially, given that [8460](https://moneyadviceservice.tpondemand.com/entity/8460) does not state the tool IDs for the English and Welsh pages? 

Or do you prefer to do this at a later date? AFAIK there's no story dealing with the tool IDs for this tool.